### PR TITLE
add docs for ci credits

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/user-guide/ci/_index.md [hunk 1]
```diff
@@ -3,30 +3,32 @@ title: "LocalStack in CI"
...
-The test reports created by your testing framework can be enriched with traces and analytics generated inside LocalStack.
+## Hypothetical CI workflow
 
-{{< figure src="localstack-in-ci.svg" alt="An example CI/CD workflow using LocalStack" width="70%">}}
+Let's assume that your team has an automated CI workflow into which you want to integrate end-to-end cloud testing with LocalStack. As an example, consider the following pipeline, which represents part of a simple CI workflow:
 
+{{< figure src="localstack-in-ci.svg" alt="An example CI/CD workflow using LocalStack" width="90%">}}
 
-## Running LocalStack in CI environments
+The CI build is triggered by pushing code to a version control repository, like GitHub. The CI runner starts LocalStack and executes the test suite. You can also use the same Infrastructure-as-Code (IaC) configuration that you use to set up AWS in your production environment to set up LocalStack in the CI environment. You can also pre-seed state into the local AWS services (e.g., DynamoDB entries or S3 files) provided by LocalStack in your CI environment via Cloud Pods.
```
Comment (whummer) [inline content/en/user-guide/ci/_index.md:22]:
```suggestion
The CI build is triggered by pushing code to a version control repository, like GitHub. The CI runner starts LocalStack and executes the test suite. You can also use the same Infrastructure-as-Code (IaC) configuration that you use to set up AWS in your production environment to set up LocalStack in the CI environment. You can also pre-seed state into the local AWS services (e.g., DynamoDB entries or S3 files) provided by LocalStack in your CI environment via [Cloud Pods](https://docs.localstack.cloud/tools/cloud-pods/). 
```

File: content/en/user-guide/ci/_index.md [hunk 2]
```diff
@@ -3,30 +3,32 @@ title: "LocalStack in CI"
...
+After a successful test run, you can execute the more expensive AWS CodeBuild pipeline for deploying your application. You can enrich the test reports created by your testing framework with traces and analytics generated inside LocalStack.
+
+## CI Credits
+
+A CI key allows you to use LocalStack in your CI environment. Every activation of a CI key consumes one build credit. LocalStack Pro offers a CI key with a limited number of CI credits to help you can start experimenting with larger CI settings. LocalStack Team is focused on using LocalStack on individual user machines, across teams, and in larger CI settings to help teams collaborate and use LocalStack extensively in CI.
+
+The Pro subscription is mainly intended for use on individual user machines. We recommend our Team plan if you plan to use LocalStack extensively, including your CI environments. For the CI environment, each subscription (both Pro and Team) comes with one extra CI key (free of charge), which allows you to use LocalStack in your CI environment. The CI key has a certain number of credits, depending on the number of individual user seats and the plan you have purchased.
```
Comment (whummer) [inline content/en/user-guide/ci/_index.md:30]:
```suggestion
The Pro subscription is mainly intended for use on individual user machines. We recommend our Team plan if you intend to use LocalStack extensively for team collaboration and in CI environments. For the CI environment, each subscription (both Pro and Team) comes with one extra CI key (included in the subscription at no extra charge), which allows you to use LocalStack in your CI environment. The CI key has a certain number of credits, depending on the number of individual user seats and the plan you have purchased.
```

File: content/en/user-guide/ci/_index.md [hunk 3]
```diff
@@ -3,30 +3,32 @@ title: "LocalStack in CI"
...
+
+## CI integrations
+
+The steps required for the integration differ slightly depending on your preferred CI platform. However, all the different options need you to retrieve your CI key from your Account settings on LocalStack App. Please refer to the relevant sections below for detailed instructions on the integration process.
```
Comment (whummer) [inline content/en/user-guide/ci/_index.md:34]:
```suggestion
The steps required for the integration differ slightly depending on your preferred CI platform. Please refer to the relevant sections below for detailed instructions on the integration process. To follow the instructions, you can first retrieve the CI key from the Account settings page in the [LocalStack Web app](https://app.localstack.cloud).
```

File: content/en/user-guide/ci/_index.md [hunk 4]
```diff
@@ -3,30 +3,32 @@ title: "LocalStack in CI"
...
 
-It is easy to run LocalStack in your CI runners.
-For some CI environments, for example Circle CI, we provide plugins that allow seamless integration of LocalStack in your workflow.
-But LocalStack can work in any CI environment, and we have several examples in the sections below.
+After a successful test run, you can execute the more expensive AWS CodeBuild pipeline for deploying your application. You can enrich the test reports created by your testing framework with traces and analytics generated inside LocalStack.
+
+## CI Credits
+
+A CI key allows you to use LocalStack in your CI environment. Every activation of a CI key consumes one build credit. LocalStack Pro offers a CI key with a limited number of CI credits to help you can start experimenting with larger CI settings. LocalStack Team is focused on using LocalStack on individual user machines, across teams, and in larger CI settings to help teams collaborate and use LocalStack extensively in CI.
```
Comment (whummer) [inline content/en/user-guide/ci/_index.md:28]:
```suggestion
A _CI key_ is a special type of API key that allows you to use LocalStack in your CI environment. Each key activation, i.e., each single startup of the LocalStack container in your CI environment, consumes one build credit. LocalStack Pro offers a CI key with a limited number of build credits to help you start experimenting with larger CI settings. LocalStack Team is focused on using LocalStack on individual user machines, across teams, and in larger CI settings to help teams collaborate and use LocalStack extensively in CI.
```

**Review by whummer — APPROVED** [review]
Nice write-up @HarshCasper ! LGTM overall, just a few nitpick suggestions. Will let @Sheg-Ge do the final review 👍

**Comment by HarshCasper** [comment]
Merging it now since it would go on the `beta` branch and we can further change things there in case details need to be updated.

**Pull request merged**